packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -86,6 +86,10 @@ export default class Month extends React.Component {
      * A callback function to execute when a week is selected.
      */
     onWeekSelect: PropTypes.func,
+    /**
+     * A callback function to execute when month component receives focus.
```

Comment:
We should document it does not take parameters

Reply:
updated [Docs](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-df786ca21c3be186c57955b173f95877)